proxy/mempool/mempool_schedule.py

```diff
@@ -408,6 +408,11 @@ def add_tx(self, tx: MPTxRequest) -> MPTxSendResult:
         sender_pool = self._get_or_create_sender_pool(tx.sender_address)
         LOG.debug(f'Got sender pool {tx.chain_id, tx.sender_address} with {sender_pool.len_tx_nonce_queue} txs')
 
+        if self.tx_cnt >= self._capacity:
```

Comment:
It is too late, the mempool is already full.
Please check the property `MPTxSchedule::min_gas_price`, it has the barrier for the increase of the gas-price.
I suggest moving the percent of mempool fullness to the config and use the percent in two places:
- here, to reject transactions with a gap
- in the `MPTxSchedule::min_gas_price` to increase the minimal gas-price.

Please, set the default value to 90% for the config option.